include/js/general.js

```diff
@@ -7718,6 +7718,21 @@ function cbdzdragFinish() {
 	document.getElementById('corebosdropzonemsg').style.display = 'none';
 }
 
+async function downloadFilesAsZip(fileName, urls) {
+	var baseurl = `index.php?module=Utilities&action=UtilitiesAjax&file=ExecuteFunctions&functiontocall=compressAndDownloadFiles&zipFileName=test.zip&filesToBeCompressed=${JSON.stringify(urls)}`;
+	fetch(baseurl)
```

Comment:
just to be sure you know that you can call this with the [ExecuteFunctions](https://github.com/tsolucio/corebos/blob/master/include/js/vtlib.js#L404) function.

Reply:
I tried it that way and converted the response into a blob but it the zip file wasn't opening. but using the fetch direcly it works
![image](https://user-images.githubusercontent.com/59079190/227511871-28eb02d9-bba2-4596-a22d-6f0b4ab5f2c4.png)